bitnami/cert-manager/templates/cainjector/rbac.yaml

```
@@ -0,0 +1,102 @@
+{{- if .Values.rbac.enabled }}
+---
+apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
+kind: Role
+metadata:
+  name: {{ include "certManager.cainjector.fullname" . }}-leaderelection
```

Trim the name to avoid overpassing k8s limits (this apply to some other manifests too):
```suggestion
  name: {{ printf "%s-leaderelection" (include "certmanager.cainjector.fullname" .) | trunc 63 | trimSuffix "-" }}
```